lib/event_sourcery/event_processing/event_stream_processor.rb

```diff
@@ -65,9 +47,13 @@ def processor_name(name = nil)
         end
 
         def process(*event_classes, &block)
-          event_classes.each do |event_class|
-            processes_events event_class.type
-            @event_handlers[event_class.type] << block
+          if event_classes.empty?
+            @all_event_handler = block
```

Comment:
If `process` is called without event classes more than once the first will be ignored, whereas with event classes they'll both be called... Shall we throw an error if `@all_event_handler` is already defined? Or make it possible to define multiple "all_event" handlers?